src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Header comments explaining the class is missing. Don't include the sub classes here.